### ISSUE TITLE
identserver.cpp - any reason it hasn't been adopted here? [$20]

### ISSUE THREAD

**Issue by md-5** [znc/znc#460]
##

**Comment by md-5** [comment]
Added some extra bounty. Would love to see this or a better identfile module in core. Once again the whole stall reconnect queue thing is annoying.

**Comment by Mikaela** [comment]
Oidentd supports masquerading, could it forward ident requests to identserver running in different port?

**Comment by md-5** [comment]
>Oidentd supports masquerading, could it forward ident requests to identserver running in different port?


If this were added as a supported module, I'd just use iptables to forward it. Pretty sure no one cares about identd for stuff other than IRC

**Comment by Mikaela** [comment]
But if same host was running ZNC for those who wish it and also gave shell access for IRC, identserver wouldn't work in that case.

**Comment by md-5** [comment]
>But if same host was running ZNC for those who wish it and also gave shell access for IRC, identserver wouldn't work in that case.

Invalid point though, a shell host wouldn't allow masquerading anyway.

**Comment by Mikaela** [comment]
> Invalid point though, a shell host wouldn't allow masquerading anyway.

I am thinking of shell host running ZNC for users who want it and also providing shell access for users who don't.

I am not sure if any of these exist though.

**Comment by DarthGandalf** [comment]
Yep, such shell/ZNC providers exist. Or at least, existed before.
